crates/core/tedge_api/src/messages.rs

```diff
@@ -498,6 +498,19 @@ impl From<SoftwareError> for Option<SoftwareModuleItem> {
     }
 }
 
+/// Payload of SoftwareList and SoftwareUpdate commands metadata
+#[derive(Debug, Clone, Default, Deserialize, Serialize, Eq, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct SoftwareCommandMetadata {
+    #[serde(skip_serializing_if = "String::is_empty")]
+    pub description: String,
```

Comment:
What's the purpose of this `description`? It's populated with an empty string for now.

Reply:
I added this since it's documented [here](https://thin-edge.github.io/thin-edge.io/references/agent/software-management/#operation-registration). However, I am reconsidering to remove it as we don't use it